src/crate/client/tests.py

```diff
@@ -56,6 +55,7 @@
 )
 from .sqlalchemy.tests import test_suite as sqlalchemy_test_suite
 from .sqlalchemy.types import ObjectArray
+from ..testing.util import datetime_now_utc_naive, date_now_utc_naive
```

Comment:
why this is relative?

Reply:
PyCharm created that import line for me. The import statements above are also relative.